django_freeradius/tests/base/test_utils.py

```diff
@@ -7,17 +7,24 @@
 class BaseTestUtils(object):
     def test_find_available_username(self):
         User = get_user_model()
-        User.objects.create(username='rohith', password='password')
-        self.assertEqual(find_available_username('rohith', []), 'rohith1')
-        User.objects.create(username='rohith1', password='password')
-        self.assertEqual(find_available_username('rohith', []), 'rohith2')
+        User.objects.create(username="rohith", password="password")
+        self.assertEqual(find_available_username("rohith", []), "rohith1")
+        User.objects.create(username="rohith1", password="password")
+        self.assertEqual(find_available_username("rohith", []), "rohith2")
 
     def test_validate_csvfile(self):
-        invalid_csv_path = self._get_path('static/test_batch_invalid.csv')
-        improper_csv_path = self._get_path('static/test_batch_improper.csv')
+        invalid_csv_path = self._get_path("static/test_batch_invalid.csv")
+        improper_csv_path = self._get_path("static/test_batch_improper.csv")
+        invalid_format_path = self._get_path("static/test_batch_invalid_format.pdf")
```

Comment:
I think this test method is becoming too big, can you please write a new test method with the assertions you have added?

Another suggestion: please keep the changes to minimum, do not edit lines that are not bugged, right now I see you're changing a lot of single quotes to double quotes, which even if it's good for consistency, adds noise and makes the work of the reviewers harder. Those kind of clean up changes should be done separately if you really want to do it (clean up is welcome in my opinion).

Reply:
@nemesisdesign ok I will try to write a completely new method for testing the validation of csv files.
Actually,my formatter was automatically doing these cleanup changes you mentioned.I will remember not to do any kind of edits in non-buggy lines!